### PR TITLE
Studies page basic rendering

### PR DESCRIPTION
**Pull request by AkashAryal**
Studies page loads all entries in db collection "studies." Trending and Recommend is not yet implemented.

### CODE REVIEW

File: client/src/components/Study.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, { ReactElement } from 'react';
+
+type P = {
```
Comment (SamChou19815) [inline client/src/components/Study.tsx]: Let's call it `Props` for consistency

File: client/src/components/Study.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  study: AppStudyWithoutID;
```
Comment (SamChou19815) [inline client/src/components/Study.tsx]: add readonly for extra strictness

File: client/src/pages/studies.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  }, []);
+
+  const getAllStudies = (): void => {
+    firebase
+      .firestore()
+      .collection('studies')
+      .onSnapshot((doc) => {
+        const data: AppStudyWithoutID[] = [];
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        doc.forEach((d: any) => data.push(d.data()));
```
Comment (SamChou19815) [inline client/src/pages/studies.tsx]: you can use `as` type cast

Comment (AkashAryal) [reply client/src/pages/studies.tsx]: What should i type cast here?

Comment (SamChou19815) [reply client/src/pages/studies.tsx]: to AppStudyWithoutID

Comment (SamChou19815) [reply client/src/pages/studies.tsx]: e.g. d.date() as AppStudyWithoutID

**Pull request merged**